src/Appwrite/Migration/Version/V21.php

```diff
@@ -148,6 +148,17 @@ private function migrateCollections(): void
                     } catch (\Throwable $th) {
                         Console::warning("'scheduleId' from {$id}: {$th->getMessage()}");
                     }
+                    break;
+                case 'files':
+                    try {
+                        /**
+                         * Increase metadata size to 7500
+                         */
+                        $this->projectDB->updateAttribute($id, 'metadata', size: 75000);
```

Comment:
Size above 65535 will cause breaking change. For simplicity of 1.6 migration, we should avoid that